spec/models/prayer_spec.rb

```diff
@@ -11,7 +11,7 @@
     it "must have a resident" do
       expect do
         build(:prayer, resident: nil).save!
-      end.to raise_error(ActiveRecord::NotNullViolation)
```

Comment:
@ittullos I noticed we've been expecting the wrong error here, this is going to give us a 500 status code on a request, we should be using a validation to prevent this error from happening.

Reply:
Gotcha!